.github/workflows/externalworkloads.yaml

```
@@ -70,18 +70,22 @@ jobs:
           echo ::set-output name=owner::${OWNER}
 
       - name: Create GCP VM
-        run: |
-          gcloud compute instances create ${{ env.vmName }} \
-            --labels "usage=${{ github.repository_owner }}-${{ github.event.repository.name }},owner=${{ steps.vars.outputs.owner }}" \
-            --zone ${{ env.zone }} \
-            --machine-type e2-custom-2-4096 \
-            --boot-disk-type pd-standard \
-            --boot-disk-size 10GB \
-            --preemptible \
-            --image-project ubuntu-os-cloud \
-            --image-family ubuntu-2004-lts \
-            --metadata hostname=${{ env.vmName }} \
-            --metadata-from-file startup-script=${{ env.vmStartupScript}}
+        uses: nick-invision/retry@v2
```

We should pin this GH Action to a SHA for better control over the dependency:
```suggestion
        uses: nick-invision/retry@45ba062d357edb3b29c4a94b456b188716f61020
```